tests/_support/Step/Acceptance/Administrator/Content.php

```diff
@@ -63,12 +63,14 @@ public function iCreateNewContent($title, $content)
 	 * Method to save an article
 	 *
 	 * @When I save an article
+	 * @When I save the category
+	 * @When I save the setting
```

Comment:
Unfortunately combining multiple annotation doesn't work here. When similar thing is working here in Login step.

Here technically `save` button is pressed and performing an action, but it is giving warning in console that Gherkin steps if missing for `@When I save the category` and `@When I save the setting`.

I noticed that is considering only first `@When I save an article` and ignoring rest of the two. I think it is because bot is not able to understand multiple `@when` for the same method.

It is giving below warning in console.

![annotation_doesnt_work](https://cloud.githubusercontent.com/assets/17275893/17641615/f6fbacb4-6144-11e6-9c82-2ab096066c39.png)